lang/lpython/files/0002-UserOverride.cmake-fix-native-optflag-for-PowerPC.patch

```diff
@@ -0,0 +1,27 @@
+From d4cda07a17b24cd5897cc50c1c4d5d4ccdb3b1e3 Mon Sep 17 00:00:00 2001
+From: Sergey Fedorov <vital.had@gmail.com>
+Date: Wed, 10 Jan 2024 12:49:54 +0800
+Subject: [PATCH] UserOverride.cmake: fix native optflag for PowerPC
+
+---
+ cmake/UserOverride.cmake | 7 ++++++-
+ 1 file changed, 6 insertions(+), 1 deletion(-)
+
+diff --git cmake/UserOverride.cmake cmake/UserOverride.cmake
+index 3139f24c8..5c5030118 100644
+--- cmake/UserOverride.cmake
++++ cmake/UserOverride.cmake
+@@ -9,7 +9,12 @@
+ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+     # g++
+     set(common "-Wall -Wextra")
+-    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 -march=native -funroll-loops -DNDEBUG")
++    if (CMAKE_SYSTEM_PROCESSOR MATCHES "ppc|powerpc")
++        set(native "-mtune=native")
++    else ()
++        set(native "-march=native")
++    endif ()
++    set(CMAKE_CXX_FLAGS_RELEASE_INIT "${common} -O3 ${native} -funroll-loops -DNDEBUG")
```

Comment:
I didn't look at the full CMake file but nothing should be compiled with `native`; that should all be stripped and only be done in a `native` variant. Similarly the correct flag is `-Os` not `-O3`

Reply:
Not really sure about optimization level (it is not my choice but of the upstream), but I can change it.